src/components/NavigationBar.js

```diff
@@ -3,6 +3,7 @@ import React, { useState } from "react";
 import RightMenu from "./NavSections/RightMenu";
 import { Drawer, Button } from "antd";
 import { HomeOutlined, MenuOutlined } from "@ant-design/icons";
+import {Link} from 'react-router-dom'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7284710)
```suggestion
import {Link} from "react-router-dom"
```